src/BlackSlope.Api.Common/Exceptions/ExceptionListItem.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BlackSlope.Api.Common.Exceptions
+{
+    public class ExceptionListItem

```

Comment:
Nit: `HandledSuggestionListItem`?

Reply:
I prefer the current name, but thank you for the suggestion